.github/workflows/eks-beta-add-on-integ-test.yml

```diff
@@ -76,6 +76,17 @@ jobs:
           role-to-assume: ${{ env.TERRAFORM_AWS_ASSUME_ROLE }}
           aws-region: us-west-2
 
+      - name: Confirm EKS Version Support
+        run: |
+          echo k8s.io/client-go version $(go list -m k8s.io/client-go)
+          echo version to test against ${{ matrix.arrays.k8sVersion }}
+          echo k8s.io/client-go version is v0.x.y matrix version is 1.x
```

Comment:
nit: this print statement is confusing unless we substitute the x with the actual versions we are comparing

Reply:
Removed